docs/docs/concepts/langgraph_control_plane.md

```diff
@@ -0,0 +1,98 @@
+# LangGraph Control Plane
+
+The term "control plane" is used broadly to refer to the Control Plane UI where users create and update [LangGraph Servers](./langgraph_server.md) (deployments) and the Control Plane APIs that support the UI experience.
+
+When a user makes an update through the Control Plane UI, the update is stored in the control plane state. The [LangGraph Data Plane](./langgraph_data_plane.md) "listener" application polls for these updates by calling the Control Plane APIs.
+
+## Control Plane UI
+
+From the Control Plane UI, you can:
+
+- View a list of outstanding deployments.
+- View details of an individual deployment.
+- Create a new deployment.
+- Update a deployment.
+- Update environment variables for a deployment.
+- View build and server logs of a deployment.
+- Delete a deployment.
+
+The Control Plane UI is embedded in [LangSmith](https://docs.smith.langchain.com/langgraph_cloud).
+
+## Control Plane API
+
+This section describes data model of the LangGraph Control Plane API. Control Plane APIs are used to create, update, and delete deployments. However, they are not publicly accessible.
```

Comment:
nit: Should APIs be pluralized or just API?

Control Plane API or Control Plane APIs? (And API is usually understood to be multiple endpoints)